proxmoxtf/resource/firewall/rules.go

```diff
@@ -166,9 +165,11 @@ func Rules() *schema.Resource {
 		MkRule: {
 			Type:        schema.TypeList,
 			Description: "List of rules",
-			Required:    true,
-			ForceNew:    true,
-			Elem:        &schema.Resource{Schema: rule},
+			Optional:    true,
```

Comment:
should this be `Optional`? what is the use case?

Reply:
Currently there is no way of saying that the firewall rule list must be empty when creating the firewall rules resources.